pages/nfts/[address].tsx

```diff
@@ -12,10 +54,9 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
 };
 
 export default function NftByAddress({ address }: { address: string }) {
-  const [queryNft, nftResponse] = useNftPageLazyQuery();
-
-  const nft = nftResponse?.data?.nft;
-  console.log('nft', nft);
+  const [queryNft, { data, loading, called }] = useNftPageLazyQuery();
```

Comment:
is it possible to move this to the server? I guess it won't be a  lazy query anymore. Hmmm

Reply:
We could but sometimes I feel the query is slow and it might be better to insta-load the page and have it resolve since we have UI placeholders.